packages/edit-post/src/components/visual-editor/test/index.native.js

```diff
@@ -63,6 +71,43 @@ describe( 'when title is focused', () => {
 			screen.getAllByLabelText( /Paragraph Block. Row 3/ )[ 0 ]
 		).toBeDefined();
 	} );
+
+	it( 'media blocks should be displayed', async () => {
+		const screen = await initializeEditor( {
+			initialHtml,
+		} );
+
+		// Focus first block
+		fireEvent.press(
+			screen.getAllByLabelText( /Paragraph Block. Row 1/ )[ 0 ]
+		);
+
+		// Focus title
+		fireEvent(
+			screen.getAllByLabelText( 'Post title. test' )[ 0 ],
+			'select'
+		);
+
+		// Focus last block
+		fireEvent.press(
+			screen.getAllByLabelText( /Paragraph Block. Row 2/ )[ 0 ]
+		);
+
+		// Assert that the media buttons are visible
+		const imageButton = screen.queryAllByTestId( 'insert-image-button' );
+		expect( imageButton ).toBeDefined();
```

Comment:
I'm wondering if we should use `toBeVisible` for these cases instead of `toBeDefined` what do you think?